db/seeds.rb

```diff
@@ -6,3 +6,4 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+User.create!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') if Rails.env.development?
```

Comment:
:clap: :clap: :heart_eyes: :heart_eyes: Really you create a seed file?? I'm amazed congrats!!! :smile: 
We have [another card](https://trello.com/c/CcftfjTx) to learn it, you can read it for more information :angel: